Packs/Cymulate/Classifiers/classifier-Cymulate_v2.json

```diff
@@ -0,0 +1,29 @@
+{
+ "name": "Cymulate_v2 - Classification",
+ "id": "Cymulate_v2",
+ "defaultIncidentType": "Cymulate",
```

Comment:
this classifier doesn't exist in the repository nor the PR, please add it if needed.